### PR TITLE
simple programs 1 to 10

### PR DESCRIPTION
**Pull request by dmurugesh**

### CODE REVIEW

File: simpleprograms/001_printarray.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+let readline = require("readline-sync");
+let size = readline.question ("Enter the No. of element to store in Array: ");
```
Comment (kuism) [inline simpleprograms/001_printarray.js]: Please write it as function

File: simpleprograms/001_printarray.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+let readline = require("readline-sync");
+const inputArray = [];
+
+const enterArrayElements = array => {
+    let size = readline.question("Enter the No. of element to store in Array: ");
+    for (i = 0; i < size; i++) {
```
Comment (kuism) [inline simpleprograms/001_printarray.js]: why i is not declared

File: simpleprograms/001_printarray.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+// Question 1 =>  Write a function to print all the elements of an array. ?
+
+let readline = require("readline-sync");
+const inputArray = [];
+
+const enterArrayElements = array => {
+    let size = readline.question("Enter the No. of element to store in Array: ");
```
Comment (NidhitheCoder) [inline simpleprograms/001_printarray.js:7]: It is better to accept inputs from outside of the function.

File: simpleprograms/002_printindexmultiplesof2.js [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+const enterArrayElements = array => {
+    let size = readline.question("Enter the No. of element to store in Array: ");
+    for (let i = 0; i < size; i++) {
+        array[i] = readline.question("Enter the Element " + (i + 1) + " : ");
+    }
+    console.log(array);
+}
+
+const printElements = array => {
+    for (let i = 0; i < array.length; i++) {
```
Comment (NidhitheCoder) [inline simpleprograms/002_printindexmultiplesof2.js]: Why you need 'if ' statement when you can able to increment i with different numbers. eg. i+=2,i=i+3 etc..

File: simpleprograms/003_replace.js [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+    let size = readline.question("Enter the No. of element to store in Array: ");
+    for (let i = 0; i < size; i++) {
+        array[i] = readline.question("Enter the Element " + (i + 1) + " : ");
+    }
+    console.log(array);
+}
+
+const replaceArray = array => {
+    for (let i = 0; i < array.length; i++) {
+        array[i] *= 0;
```
Comment (NidhitheCoder) [inline simpleprograms/003_replace.js:16]: why need * ?

File: simpleprograms/005_multiplesof3&5.js [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+    let size = readline.question("Enter the No. of element to store in Array: ");
+    for (let i = 0; i < size; i++) {
+        array[i] = readline.question("Enter the Element " + (i + 1) + " : ");
+    }
+    console.log(inputArray);
+}
+
+const printElements = array => {
+    for (let i = 0; i < array.length; i++) {
+        if (array[i] % 3 == 0 && array[i] % 5 == 0 ) {
```
Comment (NidhitheCoder) [inline simpleprograms/005_multiplesof3&5.js]: use ===

**Pull request merged**